frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -63,10 +71,20 @@ const EnvironmentAccordionBody = ({
         setStrategies(featureEnvironment?.strategies || []);
     }, [featureEnvironment?.strategies]);
 
+    useEffect(() => {
+        if (strategies.length > 50) {
+            trackEvent('many-strategies');
+        }
+    }, []);
```

Comment:
was this correct way to add tracking @kwasniew ?

Reply:
seems to be correcy